### ISSUE TITLE
`@typescript-eslint/no-unsafe-argument`/`no-unsafe-return` false positive in svelte code but not in script tag

### ISSUE THREAD

**Issue by DetachHead** [sveltejs/eslint-plugin-svelte3#197]
```svelte
<!-- asdf.svelte -->
<script lang="ts">
    export let onChange: (value: any) => void
</script>
```
```svelte
<!-- main.svelte -->
<script lang="ts" context="module">
    declare const value: any
</script>

<script lang="ts">
    import Asdf from './asdf.svelte'

    let foo: unknown
    foo = value // no errors
</script>

<!--
    Unsafe return of an `any` typed value. (@typescript-eslint/no-unsafe-return)
    Unsafe assignment of an `any` value. (@typescript-eslint/no-unsafe-assignment)
-->
<Asdf onChange={(value) => (foo = value)} />
```

since it's valid to assign `any` to `unknown`, this should not be an error. [typescript-eslint playground](https://typescript-eslint.io/play/#ts=4.9.3&sourceType=module&code=CYUwxgNghgTiAEYD2A7AzgF3gMyUgXPFCgJ4CwAUJculgEayECuKA1ikgO4rwC8OeeAHoh8DvBAwYSGNVSZ4DAF6EUTALZ1JfAUmGjJ0mEA&eslintrc=N4KABGBEBOCuA2BTAzpAXGUEKQAIBcBPABxQGNoBLY-AWhXkoDt8B6Jge1tieQEMAZolp9kySgHMmAW0Qt0URNGgdokcGAC+ITUA&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA)

**Comment by DetachHead** [comment]
this issue doesn't occur in [`eslint-plugin-svelte`](https://github.com/ota-meshi/eslint-plugin-svelte) which seems to be more actively maintained

- #184
